demo/demo.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#Test oc tool
+#Assumes User is logged in to cluster
+
+path=$1
+url=$2
+
+# Will create spring rest deployment
+cd $path/basic-nginx
+ansible-galaxy install -r requirements.yml --roles-path=galaxy
+ansible-playbook -i ./.applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e skip_manual_promotion=true -e source_code_url=$url
+
+
+# Wait a bit (5 min) for the app to be finished deployed
+sleep 300
+
+#Add Comment to a file to replicate a "change"
+echo "this is a comment" >> $path/basic-nginx/index.html
+
+#Change git dir to the one we're changing
+export GIT_DIR=$path/.git
+export GIT_WORK_TREE=$path
+
+# git commit
+git add .
+git commit -m "doing a change"
+git push origin master
```

Comment:
When I run this, I get an error message:
```
demo/demo.sh: line 18: ../container-pipelines//basic-nginx/index.html: No such file or directory
fatal: not a git repository: '../container-pipelines//.git'
fatal: not a git repository: '../container-pipelines//.git'
fatal: not a git repository: '../container-pipelines//.git'
build.build.openshift.io/basic-nginx-pipeline-4 started
```

It looks like you're not actually doing anything with the path var to change directories. I would add some handling of this like:

```
pop $path
  git add .
  git commit  -m "doing a change"
  git push origin master
popd
```